src/pages/release-notes.md

```diff
@@ -1,5 +1,11 @@
 # Release Notes
 
+:::info
```

Comment:
Looks a bit weird when displayed in rich diff with the three dots and info - I'd leave those parts out and just keep the "For more information..."